CMakeLists.txt

```diff
@@ -725,20 +725,20 @@ endif(NOT HAVE_SIZEOF_SSIZE_T)
 
 # Different sizeofs, etc.
 
-#    define CURL_SIZEOF_LONG        4
+#    define SIZEOF_LONG        4
 #    define CURL_TYPEOF_CURL_OFF_T  long long
 #    define CURL_FORMAT_CURL_OFF_T  "lld"
 #    define CURL_FORMAT_CURL_OFF_TU "llu"
 #    define CURL_FORMAT_OFF_T       "%lld"
-#    define CURL_SIZEOF_CURL_OFF_T  8
+#    define SIZEOF_CURL_OFF_T  8
 #    define CURL_SUFFIX_CURL_OFF_T  LL
 #    define CURL_SUFFIX_CURL_OFF_TU ULL
 
-set(CURL_SIZEOF_LONG ${SIZEOF_LONG})
+set(SIZEOF_LONG ${SIZEOF_LONG})
```

Comment:
Oops, thanks - will fix. (I couldn't really test the cmake build locally due to #1370)